ethereumj-core/src/main/java/org/ethereum/config/blockchain/Eip160HFConfig.java

```diff
@@ -76,8 +76,7 @@ public Constants getConstants() {
     @Override
     public boolean acceptTransactionSignature(Transaction tx) {
         // Restoring old logic. Making this through inheritance stinks too much
-        if (!tx.getSignature().validateComponents() ||
-                tx.getSignature().s.compareTo(SECP256K1N_HALF) > 0) return false;
-        return  tx.getChainId() == null || Objects.equals(getChainId(), tx.getChainId());
+        return tx.getSignature().validateComponents() &&
```

Comment:
The coverage probably comes from here https://github.com/ethereum/ethereumj/blob/develop/ethereumj-core/src/test/java/org/ethereum/jsontestsuite/suite/runners/TransactionTestRunner.java#L89 and it might cover all lines, but I'm not sure this covers all scenarios.